### DISCUSSION TITLE
Can I use form in user facing frontend with fully custom style?

### DISCUSSION THREAD

**Discussion by KarlisJ** [filamentphp/filament#4709] [Help]
I've used the Admin panel in many cases when I need a backend admin panel for the site and love the overall design and experience.

I want to use the form builder if the user-facing frontend part of the website which has a specific design that I have to achieve. 

Is there a way to apply custom styles to the form?

**Comment by danharrin** [comment]
Yes, all form components have CSS classes that you can use to style
